src/app/containers/RadioSchedule/index.stories.jsx

```diff
@@ -13,14 +13,22 @@ import { getLocalRadioScheduleEndpoint } from '#lib/utilities/getRadioSchedulesU
 
 // Currently, only these services have radio schedule data
 const validServices = [
+  'afaanoromoo',
   'afrique',
+  'amharic',
   'arabic',
+  'bengali',
+  'burmese',
+  'gahuza',
   'hausa',
+  'indonesia',
   'korean',
   'pashto',
   'persian',
   'somali',
   'swahili',
+  'tigrinya',
+  'urdu',
 ];
 
 const renderRadioScheduleContainer = service => (
```

Comment:
For the service `afaanoromoo`, the route for the radio schedule is `/afaanoromoo/bbc_oromo_radio/schedule.json`. So in this file we'll need to also pass in `radioService` of `oromoo` to `getLocalRadioScheduleEndpoint`. 
So one way of addressing this could be to instead of `validServices` being an array, having an object with `radioService` as well as `service`. 

It'd be good to display the other routes that have different service & radio services: `/persian/bbc_dari_radio/schedule.json` as well as `/indonesia/bbc_indonesian_radio/schedule.json`